library/common/config_template.cc

```diff
@@ -32,6 +32,10 @@ const char* config_template = R"(
                         base_interval: 0.25s
                         max_interval: 60s
         http_filters:
+          - name: envoy.filters.http.platform_extension
+            typed_config:
+              "@type": type.googleapis.com/envoymobile.extensions.filters.http.platform_extension.Bridging
+              name: PlatformExample
```

Comment:
It'd be great to document how this works since it'll be removed before merging. Can we open an issue to document it?